services/account_type/master_payer.yaml

```diff
@@ -10,7 +10,13 @@ Description: CloudZero Master Payer Template
 
 Parameters:
   IsMasterPayerAccount:
-    Description: Flag to indicate if this is the master payer account
+    Description: Flag to indicate if this is the master payer account, i.e. existing valid CUR
+    Type: String
+    AllowedValues:
+      - 'true'
+      - 'false'
+  IsOrganizationMasterAccount:
```

Comment:
New parameter, will create CUR if `IsOrganizationMasterAccount == true and IsMasterPayerAccount == false`